api/src/http_endpoint_v1.rs

```diff
@@ -257,6 +257,23 @@ impl EndpointHandler for TakeoverHandler {
     }
 }
 
+pub struct StartHandler {}
+impl EndpointHandler for StartHandler {
+    fn handle_request(
+        &self,
+        req: &Request,
+        kicker: &dyn Fn(ApiRequest) -> ApiResponse,
+    ) -> HttpResult {
+        match (req.method(), req.body.as_ref()) {
+            (Method::Put, None) => {
```

Comment:
`PUT` method in RESTful API design means that the operation can be executed multiple times and always have an idempotent state, is this what the API expects?

Reply:
It's expected that daemon in `READY` is trigger to `RUNNING`. Just like takeover endpoint with PUT method, it's required for daemon in certain status.

Reply:
> `PUT` method in RESTful API design means that the operation can be executed multiple times and always have an idempotent state, is this what the API expects?

It's expected that daemon in READY is trigger to RUNNING. Just like takeover endpoint with PUT method, it's required for daemon in certain status.